src/cli/cluster/base_path_proxy.js

```diff
@@ -24,9 +24,13 @@ export default class BasePathProxy {
 
     const { cert } = config.get('server.ssl');
     if (cert) {
-      this.proxyAgent = new HttpsAgent({
-        ca: readFileSync(cert)
-      });
+      const httpsAgentConfig = {};
```

Comment:
why didn't HttpAgent API do the sensible thing and call the property `allowUnauthorized`, and not have a falsy value default to `true`. ;)